### PR TITLE
FYST-162 Custom monetary fields review page update

### PR DESCRIPTION
**Pull request by tahsinaislam**
## Link to pivotal/JIRA issue
https://codeforamerica.atlassian.net/browse/FYST-162

## Is PM acceptance required? 
- Yes - don't merge until JIRA issue is accepted!

## What was done?
- The review pages were updated for NC and AZ to have the cents rounded to the nearest cent to show the clients how the values they inputed have been rounded. 

## How to test?
Product to use heroku link to check the moetary fields input and review page

## Screenshots (for visual changes)
- Before
Screenshot 2024-09-30 at 6 22 17 PM

- After
Screenshot 2024-09-30 at 6 22 02 PM

### CODE REVIEW

File: app/views/state_file/questions/az_review/edit.html.erb [hunk 1]
```diff
@@ -59,7 +59,7 @@
       <% if current_intake.az321_contributions.present? %>
         <% current_intake.az321_contributions.each do |contribution| %>
           <p class="text--bold spacing-below-5"><%=t(".az321_contribution_name", charity_name: contribution.charity_name, charity_code: contribution.charity_code) %></p>
-          <p><%= number_to_currency(contribution.amount) || t("general.none") %></p>
+          <p><%= number_to_currency(contribution.amount, precision: 2) || t("general.none") %></p>
```
Comment (arinchoi03) [inline app/views/state_file/questions/az_review/edit.html.erb:62]: if `contribution.amount` is `nil`, what is returned? wondering when `t("general.none")` is shown

**Review by arinchoi03 — COMMENTED** [review]
where is the `household_excise_credit_claimed_amount` for az intakes?

Also, does state_file1099_gs not have a review page for the columns that were updated?

noting: `tribal_wages_amount` for nc was updated in a different PR

File: app/views/state_file/questions/az_review/edit.html.erb (hunk 1 above)
Comment (tahsinaislam) [reply app/views/state_file/questions/az_review/edit.html.erb:62]: it just say "None"

Comment (arinchoi03) [reply app/views/state_file/questions/az_review/edit.html.erb:62]: oh i guess `number_to_currency(nil, precision: 2)` is returning `nil` to allow the "None" text to show up

**Review by arinchoi03 — APPROVED** [review]

**Comment by arinchoi03** [comment]
noting here that we chatted about the absence of 
State file 1099 gs
- Federal income tax withheld amount
- State income tax withheld amount

State file w2s
- State wages amount
- State income tax amount
- Local wages and tips amount
- Local income tax amount

from the respective review pages:
>basically this was the intended design because we wanted to give the clients the biggest value from each form w2 and 1099G to review so it is most easily identifiable

**Pull request merged**